test/gscraper/accounts_test.exs

```diff
@@ -0,0 +1,21 @@
+defmodule Gscraper.AccountsTest do
+  use Gscraper.DataCase
+
+  alias Gscraper.Accounts
+
+  describe "users" do
+    alias Gscraper.Accounts.User
```

Comment:
This can be move to group with the above alias

Reply:
Fixed in 7c36f2d.